### DISCUSSION TITLE
antdv4升级到v5，使用了less变量，怎么在confing.ts里面配置v4Token

### DISCUSSION THREAD

**Discussion by qc-z** [umijs/umi#10480] [Q&A] [answered]
文档也没有说得很清楚

**Comment by fz6m** [comment, marked as answer]
如果是 antd 插件，在 antd [运行时配置](https://umijs.org/docs/max/antd#%E8%BF%90%E8%A1%8C%E6%97%B6%E9%85%8D%E7%BD%AE)修改。

如果是 max 项目用了 layout 插件，需要符合 prolayout 的方式来修改主题，详见 https://procomponents.ant.design/components/layout 。
